PurchasesTests/Purchasing/OfferingsManagerTests.swift

```diff
@@ -100,24 +100,114 @@ extension OfferingsManagerTests {
         expect(obtainedOfferings).to(beNil())
     }
 
-    func testOfferingsForAppUserIDReturnsUnexpectedBackendErrorIfBadBackendRequest() {
```

Comment:
in case you're wondering, this behavior (`OfferingsForAppUserIDReturnsUnexpectedBackendErrorIfBadBackendRequest`) still happens, but the test got moved a little further down